sdlf-pipLibrary/external_wheels.py

```diff
@@ -0,0 +1,62 @@
+"""
+Upload the Glue Wheel for the listed external libraries
+
+MANDATORY args:
+1 - Auxiliary S3 bucket
+
+E.g. python3 MY_BUCKET_NAME
+"""
+
+import sys
+import json
+import urllib
+from time import sleep
+from io import BytesIO
+
+import boto3
+import botocore
+
+def add_wheels(name, info, team_name, bucket, client_s3, client_ssm):
+    url = info["url"]
+    file_name = url.rsplit(sep="/", maxsplit=1)[1]
+    print(f"file_name: {file_name}")
+    key = f"common_wheels/{team_name}/{file_name}"
```

Comment:
Could we just call this: `f"wheels/{team_name}/{file_name}"`

Reply:
updated to wheels